pkg/logging/s3/s3_test.go

```diff
@@ -193,27 +198,10 @@ func createCommandMissingServiceID() *CreateCommand {
 
 func updateCommandNoUpdates() *UpdateCommand {
 	return &UpdateCommand{
-		Base:                         common.Base{Globals: &config.Data{Client: nil}},
```

Comment:
Having these values explicitly set in the test didn't identify this when it was missing.

https://github.com/fastly/cli/pull/114/files#diff-9952ff67a00a160332b383c2329448aeR114